### PR TITLE
0009608: add bShallow argument for server-side water as well

### PR DESCRIPTION
**Pull request by patrikjuvonen**
**Mantis Bug Tracker issue:**
[9608](https://bugs.multitheftauto.com/view.php?id=9608)

**Summary:**
- Server-side code was missing the bShallow argument so it was never sent to clients;
- Pretty small and simple fix, easy to test.

### CODE REVIEW

**Comment by qaisjp** [comment]
Can someone confirm if this requires some more support for backwards compatbility?

**Comment by jushar** [comment]
Yes, it does need version checks.

After incrementing the bitstream version in version.h, it needs to be wrapped in something like this (for both the server __and__ the client):
```cpp
if (bitStream.Version() > 0xBEAF)
{
   // Do your stuff
}
```

**Comment by patrikjuvonen** [comment]
Thanks! I've now pushed a commit to increment the bitStream version and wrap the packet code to handle that.

File: Client/mods/deathmatch/logic/CPacketHandler.cpp [hunk 1]
```diff
@@ -3902,14 +3902,19 @@ void CPacketHandler::Packet_EntityAdd(NetBitStreamInterface& bitStream)
                         vecVertices[i].fX = sX;
                         vecVertices[i].fY = sY;
                     }
+
+                    bool bShallow;
```
Comment (ccw808) [inline Client/mods/deathmatch/logic/CPacketHandler.cpp]: bShallow should be initialized to a default value

Comment (patrikjuvonen) [reply Client/mods/deathmatch/logic/CPacketHandler.cpp]: Thanks. This has now been changed.

**Review by qaisjp — APPROVED** [review]

**Pull request merged**